api/src/main/java/net/kyori/adventure/audience/ForwardingAudience.java

```diff
@@ -112,6 +113,11 @@ default void showTitle(final @NotNull Title title) {
     for (final Audience audience : this.audiences()) audience.showTitle(title);
   }
 
+  @Override
+  default <T> void showTitlePart(final @NotNull TitlePart<T> titlePart, @NotNull final T content) {
```

Comment:
Perhaps `sendTitlePart`?

Reply:
Yeah I think this does make more sense, more in keeping with the rest of the API